liip/imagine-bundle/1.7/etc/routing/imagine.yaml

```diff
@@ -0,0 +1,19 @@
+liip_imagine_filter: 
```

Comment:
I agree that you could just use 

```yaml
I agree that you could just use 

```yaml
liip_imagine:
    resource: '@LiipImagineBundle/Resources/config/routing.xml'
```

Is fairly enough to add routing config with flex